src/minidump.rs

```diff
@@ -1058,13 +1065,6 @@ impl<'a> MinidumpStream<'a> for MinidumpMemoryList<'a> {
         // read memory contents for each region
         let mut regions = Vec::with_capacity(descriptors.len());
         for raw in descriptors.into_iter() {
-            if raw.memory.data_size == 0
-                || raw.memory.data_size as u64 > (u64::max_value() - raw.start_of_memory_range)
-            {
-                // Bad size.
-                // TODO: just drop this memory, keep the rest?
```

Comment:
It's probably worth addressing this at some point. I spent a lot of time fixing things in Breakpad so that we could get *some* useful data out of minidumps where parts of them were broken. Being able to read the memory list even if some of the entries are corrupt would be useful!

Reply:
I agree things should be more robust, but, I'd rather punt this for now as I have way bigger fish to fry first.

Reply:
Actually I changed my mind, this is trivial to implement and seems uncontroversial to me.